docs/Tutorials/Visualization.md

```diff
@@ -29,22 +29,17 @@ evolution, run the command:
 By default, the example input files do not produce any output. This can be
 changed by modifying the options passed to `EventsAndTriggers`:
 
-```
-EventsAndTriggers:
-  ? EveryNSlabs:
-      N: 3
-      Offset: 5
-  : - Observe:
-        VariablesToObserve: [MassDensity]
-```
-
-This will trigger an observation event for the Variable MassDensity every three
-slabs after the fifth one. A successful observation will result in the creation
-of H5 files whose names can be specified in the YAML file under the options
-`VolumeFileName` and `ReductionFileName`. One volume data file will be produced
-from each Charm++ node that is used to run the executable. Each volume data
-file will have its corresponding node number appended to its file name.
-Visualization of the volume data will be described in the next section.
+\snippet ObserveExample.yaml observe_event_trigger
+
+This will trigger an observation event for the variable Psi every
+three slabs starting with slab five. A successful observation will
+result in the creation of H5 files whose names can be specified in the
+YAML file under the options `VolumeFileName` and
+`ReductionFileName`. One volume data file will be produced from each
+Charm++ node that is used to run the executable. Each volume data file
+will have its corresponding node number appended to its file name.
+Visualization of the volume data will be described in the next
+section.
```

Comment:
I understood from our Wednesday conversation that this PR would let the user specify different triggers for observations of reduction and volume data. Could this tutorial and code snippet be updated to clarify that this is possible and also illustrate how this would be done? (I can take a guess after reading the source code, but ideally reading the source wouldn't be necessary)